book/03-git-branching/sections/rebasing.asc

```diff
@@ -231,6 +231,14 @@ The opposing point of view is that the commit history is the *story of how your
 You wouldn't publish the first draft of a book, and the manual for how to maintain your software deserves careful editing.
 This is the camp that uses tools like `rebase` and `filter-branch` to tell the story in the way that's best for future readers.
 
+[CAUTION]
+====
+`git filter-branch` is no longer the recommended way to rewrite history.
+Instead use the Python script `git filter-repo`, this script is safer and more performant.
+This script is not included with git, you will need to download and install the script yourself.
+Read the documentation for `git filter-repo` [here](https://github.com/newren/git-filter-repo/).
+====
+
```

Comment:
Same comment as above, this isn't the right place to get into this kind of detail.